src/components/Pagination/CasesPerPage.tsx

```diff
@@ -0,0 +1,49 @@
+import { SelectInput } from "@govuk-react/select"
+import { useRouter } from "next/router"
+import { createUseStyles } from "react-jss"
+
+interface Props {
+  options: number[]
+  selected: number
+  pageNum: number
+  casesPerPage: number
+}
+
+const useStyles = createUseStyles({
+  "cases-per-page-picker": {
+    width: "auto"
+  }
+})
+
+const CasesPerPage: React.FC<Props> = ({ options, selected, pageNum, casesPerPage }: Props) => {
+  const router = useRouter()
+  const classes = useStyles()
+
+  return (
+    <div className="moj-pagination__results">
+      {"View "}
+      <SelectInput
+        onChange={(event) => {
+          const newCasesPerPage = event.target.value
+
+          const firstCaseIndex = (pageNum - 1) * casesPerPage + 1
+          const newPageNum = Math.floor(firstCaseIndex / parseInt(newCasesPerPage, 10)) + 1
```

Comment:
I'm trying to wrap my head around this logic. Will the first case displayed on the page stay the same after updating the number of cases?

Reply:
Yes, that's correct, I'll add a comment to clarify